monitoring/mongodb/dashboard.py

```diff
@@ -314,7 +614,7 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     description="MongoDB stores documents in collections. Collections are analogous to tables in relational databases.",
     targets=[
         Target(
-            expr=f'sum(mongodb_dbstats_storageSize{{{DATABASE_FILTER_USER}, job=~"$job", namespace="{NAMESPACE}"}})',
+            expr="sum(" + Metrics.DBSTATS_STORAGE_SIZE(DATABASE_FILTER_USER) + ")",
```

Comment:
same for database, you should consider if a default value could be appropriate, to simplify code (which does not prevent overriding when a different filter is needed)